src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * 4;

```

Comment:
The formula for the area of a square should be side * side, not side * 4. The current implementation actually calculates the perimeter, not the area.